KarhooSDK/Service/Quote/CategoryQuoteMapper.swift

```diff
@@ -10,10 +10,14 @@ import Foundation
 
 struct CategoryQuoteMapper {
```

Comment:
Like the idea of separating the mapper from the interaction 👍 (perhaps we should consider it in Android)